proposals/0167-app-services.md

```diff
@@ -1050,6 +1062,34 @@ Core will need to parse and store received params even if they are not listed in
 
 The IVI system can enable/disable app service providers from offering their services and consumers from using them through additions to the policy table. New functional groups will be introduced for `AppServiceProvider` and `AppServiceConsumer`. Furthermore, the functional groups can become more granular to provide access to publishing/consuming of specific app service types.
 
+Proposed policy table structure:
+```
+        "app_policies": {
+            "SDLSuperApp12345": {
+                "keep_context": false,
+                "steal_focus": false,
+                "priority": "NONE",
+                "default_hmi": "NONE",
+                "groups": ["Base-4"],
+                "RequestType": [],
+                "RequestSubType": [],
+                "app_services": {
+                    "MEDIA": {
+                        "service_names": ["SDL Music", "App Music"],
+                        "handled_rpcs": [{"function_id": 41}]
+                        },
+                    "NAVIGATION": {
+                        "service_names": ["SDL Navigation", "App Navigation"],
+                        "handled_rpcs": [{"function_id": 39}]                    
+                    }
+                }
+            }
+```
+
+In the policy table, the `app_policies` section will have a new `app_services` object parameter. Each object will have the service type as the object key. `service_names` will be an array of allowed app service nick names. `handled_rpcs` will be an array of objects to with the function_id of allowed handled rpcs. `handled_rpcs` as an array of objects instead of an array of integers will allow for future expansion of the `handled_rpcs` permission item.
```

Comment:
Update to:
In the policy table, the `app_policies` section will have a new `app_services` object parameter. Each object will have the service type as the object key. `service_names` will be an array of allowed app service nick names. `handled_rpcs` will be an array of **objects with** the function_id of allowed handled rpcs. `handled_rpcs` **will be an** array of objects instead of an array of integers, **this** will allow for future expansion of the `handled_rpcs` permission item.